debian/patches/renderer-do-not-prefer-netplan.patch

```diff
@@ -21,8 +19,6 @@ Index: cloud-init/config/cloud.cfg.tmpl
     # Automatically discover the best ntp_client
     ntp_client: auto
     # Other config here will be given to the distro class and/or path classes
-Index: cloud-init/tests/unittests/test_render_cloudcfg.py
```

Comment:
I'll make sure I change my quilt config to suppress these index files so we don't have more diffs in the future